dev-docs/howtoguides/testing.md

```diff
@@ -180,24 +176,24 @@ additional token obtained from https://ubuntu.com/pro needs to be set:
 following environment variable to launch your specific  AMI instead of building
 a daily ubuntu-advantage-tools image.
 ```sh
-UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-ami tox -e behave-awspro-20.04
+UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-ami tox -e behave -- -D release=focal -D machine_types=aws.pro
```

Comment:
I don't think the `reuse_image` option exists anymore so this section can be removed (along with similar sections later on)

Reply:
Updated to remove the sections about reuse_image and the snapshots. Let me know if anything else is required in the docs!